specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/MitreCoverage.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2022-01-01-preview",
+    "x-ms-code-generation-settings": {
+      "name": "SecurityInsightsClient"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/mitreCoverage": {
+      "get": {
```

Comment:
>get [](http://example.com/codeflow?start=7&length=3)

How do these get created?

Reply:
calculated on our backend. based on observations we do